docs/pages/server-access/getting-started.mdx

```diff
@@ -0,0 +1,347 @@
+---
+title: Server Access Getting Started Guide
+description: Getting started with Teleport Server Access.
+---
+
+# Getting Started
+
+Server Access often involves managing your resources, configuring new clusters, and issuing commands through a CLI or programmatically to an API.
+
+This guide introduces some of these common scenarios and how to interact with Teleport to accomplish them:
+
+1. Creating a Teleport Auth Node cluster we'll add a Teleport SSH Node to.
+2. SSH into the cluster using Teleport.
+3. Introspecting the cluster using Teleport features.
+
+<Admonition type="tip" title="Tip">
+  This guide also demonstrates how to configure Teleport Nodes into the *Bastion* pattern so that [only a single Node can be accessed publically](https://goteleport.com/blog/ssh-bastion-host/). 
+</Admonition>
+
+<Figure
+  align="center"
+  bordered
+  caption="Teleport Bastion"
+>
+  ![Teleport Bastion](../../img/server-access/getting-started-diagram.svg)
+</Figure>
+
+## Prerequisites
+
+- Two instances of your favorite Linux environment (such as Ubuntu 20.05, CentOS 8.0-1905, or Debian 10).
+- A registered Doman Name with an available subdomain like `tele.example.com`.
+- A Two-Factor Authentication app (such as Authy or Google Authenticator).
+- Teleport (=teleport.version=) installed locally.
+
+## Step 1/4. Create a cluster
+
+1. Create two new instances of your desired Linux distribution (such as Ubuntu 20.05, CentOS 8.0-1905, or Debian 10).
+
+   The first instance will host a public-facing Auth Node `tele.example.com` that will serve as the *Bastion Host* (following the recommended *Bastion*-pattern). As such, leave the following ports open:
+
+   | Port | Service | Description |
+   | - | - | - |
+   | 3023 | Proxy | SSH port clients connect to. A proxy will forward this connection to port `#3022` on the destination Node. |
+   | 3024 | Proxy	SSH port used to create "reverse SSH tunnels" from behind-firewall environments into a trusted proxy server. |
+   | 443 | Browser | Used to access the Teleport Web UI through a browser with HTTPS. Also used to obtain an ACME TLS (SSL) certificate. |
+   | 22 | Cloud Provider | Used to initially access, configure, and provision your cloud instances. We'll configure and launch our instances then demonstrate how to use the `tsh` tool and Teleport in SSH mode thereafter. |
+
+   Add an `A` record in your domain registrar entry for the `tele` subdomain and map the first IP address to the `tele` subdomain.
+
+   <Figure
+     align="center"
+     bordered
+     caption="Subdomain and IP mapping"
+   >
+     ![Subdomain and IP mapping](../../img/server-access/subdomains.png)
+   </Figure>
+
+   Your second instance will be a private resource. These are the ports you'll want to initially open:
+
+   | Port | Service | Description |
+   | - | - | - |
+   | 3023 | Proxy | SSH port clients connect to. A proxy will forward this connection to port `#3022` on the destination Node. |
```

Comment:
Because you pointed the node to the proxy, port 3023 is no longer necessary, because in this case proxy will connect using the outbound tunnel that the node has established. You can safely remove 3023 from the proxy and it will work OK.

Reply:
Verified. Missed that when removing ports. Thanks!